src/NzbDrone.Integration.Test/Sonarr.Integration.Test.csproj

```diff
@@ -4,7 +4,7 @@
     <OutputType>Library</OutputType>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.SignalR.Client" Version="6.0.5" />
+    <PackageReference Include="Microsoft.AspNetCore.SignalR.Client" Version="6.0.15" />
```

Comment:
I think it's a safe change. All our AspNetCore packages are 6.0.15

Reply:
Make sure to bump the frontend package to match, just to maintain consistency

https://github.com/Sonarr/Sonarr/blob/6ea3d8c127eafbcf9d1b6e9338b737e91e256875/package.json#L33

Reply:
@Qstick, Nice point!